### PR TITLE
Update RestoreAction.js

### LINKED ISSUE #1047: Update method creation syntax to ES6 `RestoreAction.js`
### 🆕🐥☝ First Timers Only.

This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

[About First Timers Only](http://www.firsttimersonly.com/).

### 🤔 What you need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below.

  Once claimed we add you as contributor to this repository.

- [ ] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this issue

- [ ] 📝 **Update** the file [\src/edit/actions/RestoreAction.js](https://github.com/publiclab/Leaflet.DistortableImage/blob/main/src%2Fedit%2Factions%2FRestoreAction.js) in the `Leaflet.DistortableImage` repository (press the little pen Icon) and edit the line as shown below.

```diff
@@ -1,5 +1,5 @@
 L.RestoreAction = L.EditAction.extend({
-  initialize: function(map, overlay, options) {
+  initialize(map, overlay, options) {
     var mode = L.Utils.getNestedVal(overlay, 'editing', '_mode');
     var edited = overlay.edited;
 
@@ -14,7 +14,7 @@ L.RestoreAction = L.EditAction.extend({
     L.EditAction.prototype.initialize.call(this, map, overlay, options);
   },
 
-  addHooks: function() {
+  addHooks() {
     var ov = this._overlay;
 
     L.DomEvent.on(ov, {
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/)

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions

Leave a comment below!

This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).

### PR DESCRIPTION
**Pull request by AndroGenius-codes**
Fixes #1047  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: src/edit/actions/RestoreAction.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ L.RestoreAction = L.EditAction.extend({
     L.EditAction.prototype.initialize.call(this, map, overlay, options);
   },
 
-  addHooks: function() {
+    addHooks() {
```
Comment (TildaDares) [inline src/edit/actions/RestoreAction.js:17]:
```suggestion
  addHooks() {
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @AndroGenius-codes, feel free to click the `Commit Suggestion` button. Thank you!

**Review by TildaDares — APPROVED** [review]
Amazing work @AndroGenius-codes. Thank you!!

File: src/edit/actions/RestoreAction.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ L.RestoreAction = L.EditAction.extend({
     L.EditAction.prototype.initialize.call(this, map, overlay, options);
   },
 
-  addHooks: function() {
+addHooks() {
```
Comment (TildaDares) [inline src/edit/actions/RestoreAction.js:17]:
```suggestion
  addHooks() {
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Review by TildaDares — APPROVED** [review]

**Comment by TildaDares** [comment]
Congratulations on merging your PR @AndroGenius-codes 🎉🎉

File: src/edit/actions/RestoreAction.js (hunk 1 above)
Comment (AndroGenius-codes) [reply src/edit/actions/RestoreAction.js]: > 

Done. What next?

**Pull request merged**